src/pbt/deployment/jobs/databricks.py

```diff
@@ -287,9 +288,23 @@ def _delete_jobs(self) -> Dict[str, JobInfo]:
             if not any(job.id == job_id for job_id in all_jobs.keys())
         }
 
+    def _update_databricks_json_for_artifactory(self, job_data):
```

Comment:
This seems like a pre-processing construct. For clarity let's name it as such. For any entrypoint, this is mandatory